### ISSUE TITLE
hygrothermograph2 support?

### ISSUE THREAD

**Issue by roysbike** [hannseman/homebridge-mi-hygrothermograph#93]
Hello! I buy hygrothermograph 2  version. I adding to my homebridge . But sensor not reading from BLE 

![image](https://user-images.githubusercontent.com/2388223/74474919-82489d00-4eb7-11ea-83e4-14d248df4a37.png)

**Comment by iWayne** [comment]
When connecting the v2, I got the following logs:
```
Error: Unknown event type null
    at Scanner.onDiscover (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/lib/scanner.js:134:28)
    at Noble.emit (events.js:194:15)
    at Noble.onDiscover (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/noble.js:178:10)
    at NobleBindings.emit (events.js:189:13)
    at NobleBindings.onDiscover (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/bindings.js:174:10)
    at Gap.emit (events.js:189:13)
    at Gap.onHciLeAdvertisingReport (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/gap.js:256:10)
    at Hci.emit (events.js:189:13)
    at Hci.processLeAdvertisingReport (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/hci.js:651:10)
    at Hci.processLeMetaEvent (/usr/local/lib/node_modules/homebridge-mi-hygrothermograph/node_modules/@abandonware/noble/lib/hci-socket/hci.js:608:10)
```

**Comment by petrovi4** [comment]
It would be great if the plugin supported it.

**Comment by ulrich-berl** [comment]
quick and dirty #96

**Comment by hannseman** [comment]
Just released version `3.0.0` which brings support for LYWSD03MMC / Hygrothermograph. Please see the README for details on how to configure the `bindKey` needed to decrypt the payload sent by this sensor.

Thanks @ulrich-berl for fixing this.
